boinc-client/start.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
+mkdir -p /usr/app/boinc/locale
+mkdir -p /usr/app/boinc/slots
+
 if [[ -z $ACCOUNT_KEY ]]; then
   echo 'Account key undefined - using balena key'
 else
```

Comment:
The path in sed below should be `/usr/app/`**lib/**`account_boinc.rosetta.bakerlab.org.xml` as per new [Dockerfile.template](https://github.com/balenalabs/rosetta-at-home/pull/24/files#diff-544f4a594bf9c265a7933f12d8299987R7) from this branch.

Reply:
Also, with persistent storage I have now two account configs:
```
root@ed***:/usr/app/boinc# ls -l account*xml
-rw-r--r-- 1 root root 1195 Apr 21 20:22 account_boinc.bakerlab.org_rosetta.xml
-rw-rw-r-- 1 root root  161 Apr 22 06:48 account_boinc.rosetta.bakerlab.org.xml
root@ed***:/usr/app/boinc# grep auth account*xml
account_boinc.bakerlab.org_rosetta.xml:    <authenticator>208***</authenticator>
account_boinc.rosetta.bakerlab.org.xml:    <authenticator>214***</authenticator>
root@ed***:/usr/app/boinc#
```
The second with balena weak token is used

Reply:
@ptrm good catch, thanks! I think perhaps the original account_boinc.rosetta.bakerlab.org.xml is named incorrectly (if the filename is supposed to match the project URL), and the client is creating the second one. I have this on my devices even running the current version.

Reply:
I'm not sure if updating it in the other files is critical, the client should handle it, but let's monitor it.

Reply:
Thanks. Will the consistency among all other state and scheduled reply xmls be maintained automatically by boinc when the account file fix is applied? I tried fiddling with key replacement in both account files, but got messages about invalid auth when contacting the server, and other xmls remained intact.

One might always change the Rosetta account in webui, but I am concerned about the balena evnironment variable change consequences, especially that  it is easier to propagate the auth via balena itself.

Reply:
@ptrm I've got a couple of test devices going to try this right now, and will create an issue if I see problems - would you mind doing the same if it continues to be a problem on your end too?